CHANGELOG.md

```diff
@@ -2,6 +2,41 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-beta.17 - April 12, 2021
+## Breaking changes ⚠️
+* [Annotation plugin] Rename annotation classes(Symbol -> PointAnnotation, Circle -> CircleAnnotation,Line -> PolylineAnnotation,Fill -> PolygonAnnotation) ([#227](https://github.com/mapbox/mapbox-maps-android/pull/227))
```

Comment:
Missing renames about options classes. And please align them like that for better readability:
 - Symbol -> PointAnnotation
 -  ...

Reply:
IMHO it will be too verbose to mention all the details in changelog. For breaking changes, we will add them to the migration guide

Reply:
Migration guide is for v9 -> v10 migration while those changes affect v10 only.

Reply:
In general we have section in migration guide about annotations so we must update there as well.
But I would also provide that information here - @pengdev what are your thoughts here?

Reply:
Was it different in v9? if no I think we should state this clearly in the migration guide.
And we could also include a link to the migration guide in this entry when the migration guide is updated?

Reply:
It's different in v9